centreon/www/include/configuration/configObject/hostgroup_dependency/DB-Func.php

```
@@ -65,86 +90,191 @@ function testHostGroupDependencyCycle($childs = null)
+
+            continue;
+        }
     }
 }
 
-function multipleHostGroupDependencyInDB($dependencies = [], $nbrDup = [])
+function multipleHostGroupDependencyInDB(array $dependencies = [], array $nbrDup = [])
 {
+    global $pearDB, $centreon;
+
     foreach ($dependencies as $key => $value) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$value'. (undefined)

(UnusedLocalVariable)